src/scorecard.js

```diff
@@ -0,0 +1,73 @@
+var Scorecard = function(content) {
+  this.frameRecord = [];
+  this.totals = [];
+  this.DEFAULT_FRAMES = 10;
+  this.currentTurn = 0;
+  this.maxTurns = 20;
+  for (var i = 0; i < this.DEFAULT_FRAMES; i++) {
```

Comment:
Kudos for trying to use dependency injection here to remove the responsibility from scorecard.  However, it ends up being a confusing distraction and the two objects are so tightly coupled anyway I don't think it adds any benefit.  I'm sceptical of this approach for dependency injection anyway as scorecard is still responsible for calling `new content`, which feels neither here nor there.
Beware also unnecessarily eager-instantiating your objects.  Lazy instantiation can often lead to greater flexibility.  I.e. don't create the instances of `Frame` (`content`) until you need them.